### ISSUE TITLE
[cvpaper] CVPR2019 #72 Learning Active Contour Models for Medical Image Segmentation

### ISSUE THREAD

**Issue by kazuwjnlab** [kazuwjnlab/cvpaper#71]
## \#72 [Learning Active Contour Models for Medical Image Segmentation](http://openaccess.thecvf.com/content_CVPR_2019/papers/Chen_Learning_Active_Contour_Models_for_Medical_Image_Segmentation_CVPR_2019_paper.pdf)
Xu Chen, Bryan M. Williams, Srinivasa R. Vallabhaneni, Gabriela Czanner, Rachel Williams, Yalin Zheng
### どんな論文か？
医用画像のsegmentationを改善させるため，新たなloss functionを提案した。Cross Entropy LossやDice係数はpixel-wiseであり輪郭の形を考慮できないのが難点。しかし輪郭を最適化できるActive Contour Model(ACM)もまた速度が遅いという問題があった。本論文では高速に輪郭の最適化を実現するためAC loss functionを提案した
### 新規性
ACMを用いたsegmentationの際，これまでは最適化の過程で偏微分方程式を解かなければならず計算が遅かったが，本論文はかわりにAC loss functionを通常の深層学習の手法によって最小化すれば同様の最適化を高速に行えることを示した
### 結果
心臓MRIデータセット(ACDC 2017 Challenge)を用いて左心室/右心室/心筋のsegmentation taskを行った。AC loss functionを用いた訓練で，Cross Entropy LossよりUNet, DenseNetの性能は改善し，訓練も高速化された。また4種類のstate-of-the-art segmentation modelと比較してもよい性能であった
### その他（なぜ通ったか？等）

![thumb](https://drive.google.com/uc?export=view&id=1gjenIORBHECWglBU7YFJms1NJJVHSGSu)
segmentation, active contour model, medical image, loss function,
